applications/GeoMechanicsApplication/custom_python/geo_mechanics_python_application.cpp

```diff
@@ -62,6 +62,12 @@ PYBIND11_MODULE(KratosGeoMechanicsApplication,m)
 
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, DT_TEMPERATURE)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, NORMAL_HEAT_FLUX)
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE( m, AIR_TEMPERATURE )
```

Comment:
Indentational difference.
Should the registrations always be there or can they be conditional?

Reply:
I think this is a github issue, in CLion it looks okay:
![image](https://github.com/KratosMultiphysics/Kratos/assets/56549273/e5dbc3cc-8515-4a5f-9e86-efd491658cb0)

I would propose to look at conditional registration of variables in a new issue. I think it is a bit out of scope, since we need to figure out how to do that properly (in a way that it also works for other conditionals)